pallets/gear/src/tests.rs

```diff
@@ -6936,7 +6936,7 @@ fn resume_program_works() {
         ));
 
         // if user doesn't have enough funds the extrinsic should fail
-        let to_reserve = Balances::free_balance(USER_3);
+        let to_reserve = Balances::free_balance(USER_3) - Balances::minimum_balance();
```

Comment:
for reservation here always be minimum balance, right?

Reply:
ED is in play here. We can't reserve all free balance with the new reservation logic.